.env.example

```
@@ -3,3 +3,6 @@ CHANNEL_ID=your_channel_id_here
 NOTION_API_KEY=your_notion_integration_token_here
 NOTION_DATABASE_ID=your_notion_database_id_here
 CLAUDE_MODEL=
+# Required for `local_llm.sh --briefing` (web_search tool via Brave Search).
+# Get a Free-plan key at https://api-dashboard.search.brave.com/
+BRAVE_API_KEY=your_brave_search_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BRAVE_API_KEY key should go before the CHANNEL_ID key

(UnorderedKey)